extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/health/DataSourceHealthCheck.java

```diff
@@ -52,13 +52,15 @@ public HealthCheckResponse call() {
                 if (!valid) {
                     String data = isDefault ? "validation check failed for the default DataSource"
                             : "validation check failed for DataSource '" + dataSource.getKey() + "'";
-                    String dsName = isDefault ? "default" : dataSource.getKey();
+                    String dsName = isDefault ? "<default>" : dataSource.getKey();
```

Comment:
You can drop this entirely and use `dataSource.getKey()` directly. It will be `<default>` for the default datasource.